proto/ssn/async/v1/async.proto

```diff
@@ -106,11 +107,31 @@ message PurchaseLineCandidate {
   repeated ssn.type.PurchaseLineCandidate purchase_line_candidate = 1;
 }
 
+message LineCandidate {
+  repeated ssn.type.LineCandidate line_candidates = 1;
+}
+
+message AnswerCandidate {
+  repeated ssn.type.AnswerCandidate answers = 1;
+}
+
+message PageText {
+  repeated ssn.type.PageText page_text = 1;
+}
+
+message TextAnnotation {
+  repeated ssn.type.TextAnnotation text_annotation = 1;
+}
+
 message Annotation {
   string feature = 1; // feature name e.g. "TOTAL_INCL_VAT"
   oneof candidates {
     Candidate candidate = 2; // candidate contains a page number
     PurchaseLineCandidate purchase_line_candidate = 3;
+    AnswerCandidate answer_candidates = 4;
+    LineCandidate line_candidates = 5;
+    PageText page_text = 6;
+    TextAnnotation text_annotation = 7;
```

Comment:
Why double nesting?

Reply:
because of repeated, you cannot do one of and define repeated type right away, you just have to double nest